apps/condo/domains/user/schema/SendMessageToSupportService.js

```diff
@@ -0,0 +1,122 @@
+/**
+ * Generated by `createservice user.SendMessageToSupportService`
+ */
+
+const { GQLCustomSchema } = require('@core/keystone/schema')
+const access = require('@condo/domains/user/access/SendMessageToSupportService')
+const { sendMessage } = require('@condo/domains/notification/utils/serverSchema')
+const { MESSAGE_FORWARDED_TO_SUPPORT } = require('@condo/domains/notification/constants/constants')
+const { SUPPORT_EMAIL } = require('@condo/domains/common/constants/requisites')
+const { get } = require('lodash')
+const { LOCALES } = require('@condo/domains/common/constants/locale')
+const { Resident } = require('@condo/domains/resident/utils/serverSchema')
+const { Organization } = require('@condo/domains/organization/utils/serverSchema')
+const FileAdapter = require('@condo/domains/common/utils/fileAdapter')
+const { v4: uuid } = require('uuid')
+const dayjs = require('dayjs')
+
+const SEND_MESSAGE_TO_SUPPORT_ATTACHMENTS_FILE_FOLDER_NAME = 'forwarded-emails-attachments'
+const fileAdapter = new FileAdapter(SEND_MESSAGE_TO_SUPPORT_ATTACHMENTS_FILE_FOLDER_NAME)
+
+const SendMessageToSupportService = new GQLCustomSchema('SendMessageToSupportService', {
+    types: [
+        {
+            access: true,
+            type: `enum SendMessageToSupportLang { ${Object.keys(LOCALES).join(' ')} }`,
+        },
+        {
+            access: true,
+            type: 'input SendMessageToSupportInput { dv: Int!, sender: JSON!, text: String!, email: String, attachments: [Upload], os: String!, appVersion: String!, lang: SendMessageToSupportLang!, meta: JSON! }',
```

Comment:
may be emailFrom?
```suggestion
            type: 'input SendMessageToSupportInput { dv: Int!, sender: JSON!, text: String!, emailFrom: String, attachments: [Upload], os: String!, appVersion: String!, lang: SendMessageToSupportLang!, meta: JSON! }',
```

Reply:
From the point of view of mobile, it's just an email provided by the user. But from point of view of the sending email, this is emailFrom. These points of view were reflected as `{ ..., email: emailFrom, ... }` in the muation.

Reply:
You call the action `sendMessageToSupport` with `email` and you need some extra context to understand the reason for this field. Is this a user email or a support email? It's better to add some context here for users whoos need to understand how to use the API. They don't have code context to understand what's going on.

Reply:
Done. `email` renamed to `emailFrom`